corehq/apps/ota/views.py

```diff
@@ -32,6 +42,70 @@ def restore(request, domain, app_id=None):
     return get_restore_response(domain, couch_user, app_id, **get_restore_params(request))
 
 
+@json_error
+@login_or_digest_or_basic_or_apikey()
+def search(request, domain):
+    """
+    Accepts search criteria as GET params, e.g. "https://www.commcarehq.org/a/domain/phone/search/?a=b&c=d"
+    Returns results as a fixture with the same structure as a casedb instance.
+    """
+    criteria = dict(request.GET)
+    try:
+        case_type = criteria.pop('case_type')
+    except KeyError:
+        return HttpResponse('Search request must specify case type', status=400)
+
+    search_es = CaseSearchES()
+    search_es = search_es.domain(domain)
+    search_es = search_es.case_type(case_type)
```

Comment:
just thought of this, but we should currently limit this query to return a maximum number of results (maybe 10 for now?)
e.g: `search_es.size(MAX_CASE_SEARCH_RESULTS)`